code/game/objects/items/devices/multitool.dm

```diff
@@ -76,7 +75,8 @@
 	track_delay = 5
 
 /obj/item/device/multitool/ai_detect/admin/multitool_detect()
-	for(var/mob/J in range(rangewarning,src))
+	var/turf/our_turf = get_turf(src)
+	for(var/mob/J in range(rangewarning,our_turf))

```

Comment:
Use ``urange()`` for ranges of 9 or higher, it's faster and it already does the get_turf() for you.